amaranth-chrome-ext/manifest.json

```diff
@@ -5,7 +5,13 @@
     "content_scripts": [
 	    {
 		    "matches": ["<all_urls>"],
-		    "js": ["lib/tf.min.js", "src/content.js"]
+		    "js": [
+                "lib/tf.min.js",
```

Comment:
This indent looks slightly off? Can you fix?

Reply:
Not sure how the indentation got messed up here, looks correct on my machine. I'll look into it!

Reply:
Fixed in https://github.com/googleinterns/amaranth/pull/44/commits/944a75c358aea037431a4bfd29bc9be368cf22e0